wandb_logger.py

```diff
@@ -77,5 +76,5 @@ def save_params(
     """
     wandb.log({
         "params": {
-            name: Tsor(param) for (name, param) in model.named_parameters()},
+            name: param.cpu() for (name, param) in model.named_parameters()}
```

Comment:
Cast param.cpu() to Tsor, as parameters are not serializable (see previous version of the file).

Reply:
Tsor cast causes an error when run on the GPU. modified to explicit CPU tensor.
Also, model parameters should not serialized to JSON. Added new method to upload to wandb a file.
Model file is created with torch.save(), which uses pickle.

Reply:
> Tsor cast causes an error when run on the GPU. modified to explicit CPU tensor.
> Also, model parameters should not serialized to JSON. Added new method to upload to wandb a file.
> Model file is created with torch.save(), which uses pickle.

ok. Indeed it is better this way.